src/Controller/Back/SignalementFileController.php

```diff
@@ -144,4 +144,58 @@ public function deleteFileSignalement(
 
         return $this->json(['response' => 'error'], 400);
     }
+
+    /**
+     * @throws FilesystemException
+     */
+    #[Route('/{uuid}/file/edit', name: 'back_signalement_edit_file')]
+    public function editFileSignalement(
+        Signalement $signalement,
+        Request $request,
+        FileRepository $fileRepository,
+        EntityManagerInterface $entityManager,
+    ): RedirectResponse {
+        $this->denyAccessUnlessGranted('FILE_EDIT', $signalement);
```

Comment:
Je pense que ce Voter devrait être sur l'entité `File` pour faire le même contrôle que dans les templates ou il devrait être utilisé en remplacement. Je parle de ce controle : 
`{% if canEditSignalement and isNewFormEnabled and isDocumentsEnabled and (is_granted('ROLE_ADMIN_TERRITORY') or is_granted('ROLE_ADMIN') or photo.uploadedBy.id is same as(app.user.id)) %}`

En tout cas il y'a une inconsistance entre les deux contrôles

Reply:
ok c'est fait. Je l'ai fait aussi pour le FILE_DELETE, c'était un peu plus compliqué